.travis.yml

```diff
@@ -17,9 +18,14 @@ addons:
 gemfile:
   - gemfiles/4.2.gemfile
   - gemfiles/5.0.gemfile
+  - gemfiles/6.0.gemfile
 
 matrix:
   fast_finish: true
   exclude:
     - gemfile: gemfiles/5.0.gemfile
       rvm: 2.1
+    - gemfile: gemfiles/4.2.gemfile
+      rvm: 3.0
+    - gemfile: gemfiles/5.0.gemfile
+      rvm: 3.0
```

Comment:
Rails <6.0 does not support Ruby 3, because those Rails versions [are locked to `sqlite3` <1.4](https://github.com/rails/rails/blob/v5.2.4.4/Gemfile#L139).